.env.dist

```diff
@@ -0,0 +1,13 @@
+PORT=3000
+AUTHORIZED_USER_EMAILS=kevin.masseix@beta.gouv.fr
```

Comment:
Sounds like a breach to have this email here in plain text. This is a public repo